### PR TITLE
qemu.tests: make device property support both old and new qemu

### PR DESCRIPTION
**Pull request by xutian**
In old version qemu, qtree use on/off for bool type property,
Now it use true./false instead. Update code to make this case
support both old and new version qemu output.

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1158059

File: qemu/tests/device_bit_check.py [hunk 1]
```diff
@@ -58,11 +58,16 @@ def run(test, params, env):
             raise error.TestError("Can't get device info from qtree result.")
 
         for index, option in enumerate(options):
-            option_value = re.findall("%s\s+=\s+(\w+)" % option, dev_info[0])
+            option_regex = "%s\s+=\s+(\w+)" % option
+            option_value = re.findall(option_regex, dev_info[0], re.M)
             if not option_value:
+                logging.info("dev info in qtree: %s" % dev_info[0])
```
Comment (ypu) [inline qemu/tests/device_bit_check.py]: Better to put this into debug log. or directly put it in the error msg

File: qemu/tests/device_bit_check.py [hunk 2]
```diff
@@ -77,6 +82,9 @@ def run(test, params, env):
                 bitstr = re.findall("[01]+", bitstr)[-1]
 
                 if bitstr[int(options_offset[index])] != properties[index]:
+                    msg = "bit string in guest: %s" % bitstr
+                    msg += "expect bit string: %s" % properties[index]
+                    logging.info(msg)
```
Comment (ypu) [inline qemu/tests/device_bit_check.py]: the same above

File: qemu/tests/device_bit_check.py [hunk 3]
```diff
@@ -58,11 +58,16 @@ def run(test, params, env):
...
+            option_value = re.findall(option_regex, dev_info[0], re.M)
             if not option_value:
+                logging.info("dev info in qtree: %s" % dev_info[0])
                 raise error.TestError("Can't get the property info from qtree"
                                       " result")
-            if option_value[0] != convert_dict[properties[index]]:
+            if option_value[0] not in convert_dict[properties[index]]:
+                msg = "'%s' value get '%s', " % (option, option_value)
+                msg += "expect value '%s'" % convert_dict[properties[index]]
+                logging.info(msg)
```
Comment (ypu) [inline qemu/tests/device_bit_check.py]: The same as above.

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by ypu** [comment]
Thanks @xutian @spcui and @suqinhuang will merge it.

**Pull request merged**